cisco_aci/datadog_checks/cisco_aci/api.py

```diff
@@ -58,7 +58,7 @@ def make_request(self, path):
             cookie = self.apic_cookie
         elif self.cert_key:
             payload = 'GET{}'.format(path)
-            signature = self.cert_key.sign(payload, padding.PKCS1v15(), hashes.SHA256())
+            signature = self.cert_key.sign(ensure_bytes(payload), padding.PKCS1v15(), hashes.SHA256())
```

Comment:
Note to reviewers: This line was failing for Python 3. Seems it's currently not working for this case: cert login + python3.